crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -1419,6 +1419,22 @@ impl EmptyChunk {
     }
 }
 
+/// The raw representation of a linked chunk, as persisted in storage.
+#[derive(Debug)]
+pub struct RawLinkedChunk<Item, Gap> {
+    /// Content section of the linked chunk.
+    pub content: ChunkContent<Item, Gap>,
+
+    /// Link to the previous chunk, via its identifier.
+    pub previous: Option<ChunkIdentifier>,
+
+    /// Current chunk's identifier.
+    pub id: ChunkIdentifier,
```

Comment:
To match the naming of `Chunk`, can we rename `id` to `identifier` please?

Reply:
I'd rather go the opposite way, for the sake of conciseness, and rename `identifier` to `id`, to be honest :P but consistency is king, so I'll use `identifier` here :+1: